unstructured/cleaners/core.py

```diff
@@ -139,14 +139,14 @@ def clean_trailing_punctuation(text: str) -> str:
     return text.strip().rstrip(".,:;")
 
 
-def replace_mime_encodings(text: str) -> str:
+def replace_mime_encodings(text: str, encoding: str = "utf-8") -> str:
     """Replaces MIME encodings with their UTF-8 equivalent characters.
```

Comment:
Should probably update the docstring.

Reply:
Thanks. Updated the doc string and added a test.